### PR TITLE
Added swipe gesture to itemdetailview

### LINKED ISSUE #135: Swipe to go back from detail view to listing
### Acceptance Criteria
when I'm on a detailed entry, if I swipe from the left edge of the screen to the right I expect to navigate back "up"/back to the listing view

### PR DESCRIPTION
**Pull request by sashei**
resolves #135

### CODE REVIEW

File: lockbox-ios/View/ItemDetailView.swift [hunk 1]
```diff
@@ -91,6 +92,14 @@ extension ItemDetailView {
         leftButton.rx.tap
                 .bind(to: presenter.onCancel)
                 .disposed(by: self.disposeBag)
+
+        self.view.rx.swipeGesture(.right)
```
Comment (linuxwolf) [inline lockbox-ios/View/ItemDetailView.swift]: 1. Does this need a test to verify the expected behavior happens?
2. How possible is it to test such gestures?

**Comment by devinreams** [comment]
I like it! But I have one.. mostly I don’t enjoy it automatically snapping immediately. I think lot of apps feel more “natural” when the slide follows the thumb for a little bit of travel (20% over the screen). 

I’m not the UX expert but just wanted to share my quick thought

**Comment by sashei** [comment]
@devinreams explored a little more, let me know what you think of the more recent build.

@linuxwolf Cocoa doesn't expose the methods for calling gesture recognizer target-actions programmatically, making them heinous to test. This is a better candidate for a UI test down the line than one in code IMO, but I could spend the time to get a programmatic test written if you'd like to see that.

**Comment by linuxwolf** [comment]
> @linuxwolf Cocoa doesn't expose the methods for calling gesture recognizer target-actions programmatically, making them heinous to test. This is a better candidate for a UI test down the line than one in code IMO, but I could spend the time to get a programmatic test written if you'd like to see that.

@sashei  "not feasible" is a perfectly fine and justifiable reason.

**Pull request merged**